gatsby/content/projects/bridges/mx-puppet-discord.mdx

```diff
@@ -25,15 +25,16 @@ bridges: Discord
 &#10003; Formatted Messages  
```

Comment:
This project has moved to https://gitlab.com/mx-puppet/discord/mx-puppet-discord

We need to update the `repo` field above.

Reply:
This was updated ⏩